fineract-db/adhoq.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE `m_adhoc` (
```

Comment:
This migration script should be added under fineract-provider\src\main\resources\sql\migrations\core_db. and check the existing migration scripts for filename format